dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -0,0 +1,129 @@
+SUMMARY="A powerful data analysis library for Python"
+DESCRIPTION="Pandas is a Python package providing fast, flexible \
+and expressive data structures designed to make working with \
+\"relational\" or \"labeled\" data both easy and intuitive. \
+It aims to be the fundamental high-level building block for \
+doing practical, real world data analysis in Python."
+HOMEPAGE="https://pandas.pydata.org/"
+COPYRIGHT="2008-2012 AQR Capital Management, LLC
+	Lambda Foundry, Inc.
+	PyData Development Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/3e/56/82c4d4c049294f87ebd05b65fdcbc9ed68bd23fb0a7e4469caf9a75d199f/pandas-0.23.3.tar.gz"
```

Comment:
Oops, I missed this one:
```diff
-SOURCE_URI="https://files.pythonhosted.org/packages/3e/56/82c4d4c049294f87ebd05b65fdcbc9ed68bd23fb0a7e4469caf9a75d199f/pandas-0.23.3.tar.gz"
+SOURCE_URI="https://files.pythonhosted.org/packages/source/p/pandas/pandas-$portVersion.tar.gz"
```
(Both are correct because the "new" URL redirects to the "old" one.)